.github/workflows/tests.yml

```diff
@@ -23,12 +23,12 @@ jobs:
       matrix:
         # We can run tests against any Lucee version, but we can only compile against Lucee releases available on mvnrepository.com.
         lucee_version: [ "5" ]
-        java_version: [ "8", "11", "17" ]
+        java_version: [ "8", "11" ]
         experimental: [ false ]
         include:
           - lucee_version: "6"
             java_version: "21"
-            experimental: true
+            experimental: false
```

Comment:
Ah - I see where you're going here. I would instead add the "6" version string to the `lucee_version` array on line 25, then change this block to run experimental tests on Lucee 7.

Reply:
Yeah, fair.